k2/python/csrc/torch/v2/ragged_any.h

```diff
@@ -39,7 +39,7 @@ namespace k2 {
  Ragged<Any> unifies Ragged<int32_t>, Ragged<float>, and Ragged<double>
  so that Python only sees Ragged<Any>.
 */
-struct RaggedAny {
+struct __attribute__((__visibility__("default"))) RaggedAny {
```

Comment:
Please don't use compiler-specific attributes here.

Reply:
Will fix in another PR.